### ISSUE TITLE
NO concern here

### ISSUE THREAD

**Issue by mightydanp** [GregTechCE/GregTech#1071]
just pointing this out but there hasn't been an update since jan 22nd  :)

**Comment by LAGIdiot** [comment]
Thank you for reaching to us and creating a new issue (even if it has 0 value IMHO).

You probably haven't noticed (otherwise you wouldn't have created this issue) but there is some development going on right now. We have some new features to introduce and we've fixed quite a lot of bugs.

If you are interested in more than pointing out we don't have any new updates ready, you can help out by testing.

**Comment by mightydanp** [comment]
Awesome since you are in development if you dont mind can you guys start releasing test versions. This is a good way for people that dont know how to compile the src the ability to test and find bugs.

**Comment by LAGIdiot** [comment]
I don't think it is good idea to post development versions for public use as they may not be as stable as we would like to.
If you are really interested in testing feel free to stop at GTCE Discord and will point you to our Gitlab build pipeline where you can grab latest build.
